omero/developers/whatsnew.txt

```diff
@@ -7,7 +7,9 @@ What's new for OMERO 5.1
   (currently slated to be 5.2). Most notably for development purposes, we are
   deprecating support for Java 1.6 and Python 2.6.
 - :doc:`Model/Units` introduces the new Units-types which are available
-  for use from all model objects.
+  for use from :doc:`Model Objects <Model/EveryObject>` (note that this
```

Comment:
This can wait til next week if needed, I'm just working through my list and opening PRs as I get to things!